sources/packages/backend/libs/integrations/src/esdc-integration/fed-restriction-integration/fed-restriction.processing.service.ts

```diff
@@ -77,48 +77,41 @@ export class FedRestrictionProcessingService {
           await this.integrationService.archiveFile(remoteFilePath);
         } catch (error) {
           const logMessage = `Error while archiving federal restrictions file: ${remoteFilePath}`;
-          result.errorsSummary.push(logMessage);
-          result.errorsSummary.push(parseJSONError(error));
+          processSummary.error(logMessage, error);
           this.logger.error(logMessage, error);
         }
       }
     } else {
-      result = new ProcessSFTPResponseResult();
-      result.processSummary.push(
-        "No files found to be processed at this time.",
-      );
+      processSummary.info("No files found to be processed at this time.");
     }
-
-    return result;
   }
 
   /**
    * Process all the federal restrictions records in the file.
    * @param remoteFilePath remote file to be processed.
    * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param processSummary process summary for logging.
    * @returns result of the processing, summary and errors.
    */
   private async processAllRestrictions(
     remoteFilePath: string,
     auditUserId: number,
-  ): Promise<ProcessSFTPResponseResult> {
-    const result = new ProcessSFTPResponseResult();
-    result.processSummary.push(`Processing file ${remoteFilePath}.`);
-
-    let downloadResult: FedRestrictionFileRecord[];
+    processSummary: ProcessSummary,
+  ): Promise<void> {
+    processSummary.info(`Processing file ${remoteFilePath}.`);
     this.logger.log(`Starting download of file ${remoteFilePath}.`);
```

Comment:
https://github.com/bcgov/SIMS/pull/4135#discussion_r1890896582

Reply:
To be clear, I did not remove them all because they were serving different purposes and I did not consider it as part of the immediate refactor. I can explain further if needed. I will remove this also.